### PR TITLE
Add nyanamoli/mn18.html

### PR DESCRIPTION
**Pull request by buddhist-uni**
This is the first PR in what may become a series adding some of Venerable Nyanamoli's MN translations from [A Treasury of the Buddha's Words](https://archive.org/details/a-treasury-of-the-buddhas-words_202305).

Please take extra care to review this one as it's my first legacy text and the future files will use this one as a template, especially the `footer` and name conventions.

I kept the paragraph numbering the same as in the original and thus called them "tbw" numbers. Some of Ven Nyanamoli's "paragraphs" contained carriage returns within them. I've coded these up as `<br>` (with no additional whitespace). Is that correct?

The book inexplicably uses single quotes for top level quotes and double quotes for nested quotes. I've reversed that decision and also removed hyphens that were added across line-breaks (and replaced `...`s with `…`) but have otherwise left the text's (poor) punctuation alone, opting to only fix a single typo where an extra "the" had crept in before a pronoun.

cc Bhante @sujato and @ccronje for review 😃

### CODE REVIEW

**Comment by thesunshade** [comment]
> inexplicably uses single quotes for top level quotes and double quotes for nested quotes.

That's British style if I'm not mistaken.

I think it's optional, but there is a span class for the added text. Could just be put around everything in parens.

Thanks for adding this sutta!

**Comment by buddhist-uni** [comment]
Thanks Bhante! Add spans added ✅

**Comment by sujato** [comment]
Looking good, congrats. Just a few changes:

- for the reference numbers, use `nya`, these will presumably be the same as those used by Bodhi. Maybe there might be some slight differences, never mind, but generally we try to keep the number of different editions minimal.
- And no, don't use `<br>` tags except for verse. Just mark them up as a paragraph. 
- As to the quote style, yes, better keep the double then single style for consistency within SC.

**Comment by buddhist-uni** [comment]
Okay. Updated 😄  How does it look now?

**Comment by sujato** [comment]
Great, let's go!

File: html_text/en/pli/sutta/mn/nyanamoli/mn18.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset='UTF-8'>
+<meta name='author' content='Nyanamoli Thera'>
+<title></title>
```
Comment (buddhist-uni) [inline html_text/en/pli/sutta/mn/nyanamoli/mn18.html:6]: Does it need a `<title>`, Bhante @sujato?

**Pull request merged**